cacerts/execd.go

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
I noticed this pattern and ignored it b/c I wanted to unexported functions like https://github.com/paketo-buildpacks/ca-certificates/pull/1/commits/79b9e38e4d6230b1745ff59ee76c759830c9fad8#diff-313fab271285a81f27a170c3d599f1e86c757acd469d3479e765d0d1d1160174R31

This could be done with a little reorg (pulling out paths in main) but given the helper and the build for this buildpack are doing almost identical things (and require the same imports) it made sense to me that they would live in the same package.

Reply:
Fair enough.  I'm concerned though about the unit testing implications of this being an un-exported function.  And once you take that into account, keeping them together often doesn't matter.